core/api.go

```diff
@@ -304,10 +304,29 @@ type LoadBalancer interface {
 	Next() Member
 }
 
-// HazelcastJSON is a wrapper for json byte array.
+// HazelcastJSONValue is a wrapper for json byte array.
 // JsonString should be a valid json string.
-type HazelcastJSON struct {
-	JSONString []byte
+type HazelcastJSONValue struct {
+	jsonString []byte
+}
+
+func CreateHazelcastJSONValueFromString(jsonString string) *HazelcastJSONValue {
```

Comment:
this belongs to the API, godoc should be added. Normally linter should fail saying something like public `CreateHazelcastJSONValueFromString ` should have a comment..., there is a bug with the linter or its configuration I think.